Platform/ElkhartlakeBoardPkg/AcpiTables/Dsdt/HdaSlaveDevices.asl

```diff
@@ -40,7 +40,7 @@ Scope(\_SB.PC00.HDAS.SNDW)
     Method (_ADR, 0) {
       Return (V_HDA_SNDW_LINK_ID_0 | V_HDA_SNDW_DID_REALTEK_ALC701)
     }
-    Include ("HdaSoundWireCodec.asl")
+    //Include ("HdaSoundWireCodec.asl")
```

Comment:
let us remove the commented code.

Reply:
Submitted patch to https://github.com/slimbootloader/slimbootloader/pull/1432